src/main/resources/messages/LibertyBundles.properties

```
@@ -51,6 +51,12 @@ liberty.project.file.selection.dialog.message=Select Liberty project for {0}
 liberty.action.selection.dialog.title=Liberty action
 liberty.action.selection.dialog.message=Select Liberty action to run
 
+#Messages for Checking javaHome
+javaHome.is.null=Unable to launch the {0} language server. Could not resolve the java home system property. Make sure the boot Java runtime of the IDE is pointing to a valid JDK of Java {1} or later. Change the boot Java runtime of the IDE as documented here <a href="https://www.jetbrains.com/help/idea/switching-boot-jdk.html">https://www.jetbrains.com/help/idea/switching-boot-jdk.html</a>
```

I think I may have originally wrote this sentence... I think this wording is better:
```suggestion
javaHome.is.null=Unable to launch the {0} language server. Could not resolve the java home system property. Make sure the boot Java runtime of the IDE is set to a valid JDK of Java {1} or later. Change the boot Java runtime of the IDE as documented here <a href="https://www.jetbrains.com/help/idea/switching-boot-jdk.html">https://www.jetbrains.com/help/idea/switching-boot-jdk.html</a>
```